app/templates/codelists-management/index.hbs

```diff
@@ -17,15 +17,13 @@
         <AuToolbarGroup class="au-c-toolbar__group--center">
           {{! Copied from the AuDataTable::TextSearch component since that doesn't support calling actions on input}}
           <div class="au-c-data-table__search codelist-label-search">
-            <input
-              value={{this.label}}
+            <AuInput
```

Comment:
FYI, switching to AuInput only silenced the linting error since the rule doesn't detect custom input components. 😄. Technically the input _should_ still receive a label for accessibility reasons but I'm not sure how important that is here since it didn't have them before 😅.